airbyte-test-utils/src/main/java/io/airbyte/test/utils/AirbyteAcceptanceTestHarness.java

```diff
@@ -727,6 +727,7 @@ public static void waitForSuccessfulJob(final JobsApi jobsApi, final JobRead ori
       }
     }
     assertEquals(JobStatus.SUCCEEDED, job.getStatus());
+    Thread.sleep(10000);
```

Comment:
@benmoriceau Is 10 seconds really needed here to make the tests pass consistently? I was thinking that we would only need maybe a few hundred milliseconds of wait after a successful job is seen. Having 10 seconds of wait time here will slow down all of our acceptance tests somewhat significantly, making our builds take a lot longer

Reply:
It looks like the failure that happened after you increased this to 5 seconds was just one of those `header parser received no bytes` issues, which is a separate problem.

So I think we should be able to set this sleep to a much lower value, something like `Thread.sleep(200)`